### PR TITLE
[cypher-builder] Add and export all math operators

### PR DESCRIPTION
**Pull request by jbanety**
# Description

Exports `MathOp` as `Math` in the entry point and add all math operators to the type `MathOperator`.

## Complexity

Complexity: Low

# Issue

N/A

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] TCK tests have been updated
- [ ] Integration tests have been updated
- [ ] Example applications have been updated
- [ ] New files have copyright header
- [X] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by darrellwarde** [comment]
> [CLA assistant check](https://cla-assistant.io/neo4j/graphql?pullRequest=2960) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/neo4j/graphql?pullRequest=2960) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/neo4j/graphql?pullRequest=2960) it.

@jbanety, thank you for accepting our CLA via the manual email. Apologies for the inconvenience, but we're trialling a new CLA system right now, so would you also mind accepting by following the instructions after clicking on the amber button in the above message? Thanks!

File: packages/cypher-builder/src/expressions/operations/math.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ import type { Expr } from "../../types";
 import type { CypherEnvironment } from "../../Environment";
 import { CypherASTNode } from "../../CypherASTNode";
 
-type MathOperator = "+" | "-";
+type MathOperator = "+" | "-" | "*", "/", "%", "^";
```
Comment (MacondoExpress) [inline packages/cypher-builder/src/expressions/operations/math.ts:24]:
```suggestion
type MathOperator = "+" | "-" | "*" | "/" | "%" | "^";
```

**Comment by jbanety** [comment]
> > [CLA assistant check](https://cla-assistant.io/neo4j/graphql?pullRequest=2960) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/neo4j/graphql?pullRequest=2960) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/neo4j/graphql?pullRequest=2960) it.
> 
> @jbanety, thank you for accepting our CLA via the manual email. Apologies for the inconvenience, but we're trialling a new CLA system right now, so would you also mind accepting by following the instructions after clicking on the amber button in the above message? Thanks!

Hi @darrellwarde,
I'm trying to do so but it doesn't work...

File: packages/cypher-builder/src/Cypher.ts [hunk 2]
```diff
@@ -77,7 +77,7 @@ export {
     endsWith,
     matches,
 } from "./expressions/operations/comparison";
-export { plus, minus } from "./expressions/operations/math";
+export { MathOp as Math, plus, minus } from "./expressions/operations/math";
```
Comment (MacondoExpress) [inline packages/cypher-builder/src/Cypher.ts:80]:
```suggestion
export { plus, minus, multiplication, division, modulo, exponentiation } from "./expressions/operations/math";
```

**Review by MacondoExpress — COMMENTED** [review]
Thank you for your contribution,
It's nice to see an early cypher-builder user! 
I would suggest exporting the functions multiplication/division/modulo/exponentiation functions that wraps the generic MathOp class as we currently do for plus and minus!

**Comment by MacondoExpress** [comment]
Hope you do not mind, I opened a PR that refactors the Math operators with the proper tests, linting fixes, etc, as well as the string concatenation.
Everything in this PR should be covered by https://github.com/neo4j/graphql/pull/2971 .
Thank you again for the effort!

**Comment by MacondoExpress** [comment]
Hi @jbanety, Sorry for the long wait, we merged this on dev https://github.com/neo4j/graphql/pull/315 and it will available from the next release!

**Comment by jbanety** [comment]
Nice to hear !
I'm already using this lib in production and I'll make an issue/PR toi propose some changes

**Pull request closed without merging**